petclinic-frontend/src/features/customers/components/UpdatePetForm.tsx

```diff
@@ -146,13 +196,47 @@ const UpdatePetForm: React.FC = (): JSX.Element => {
         />
         {errors.weight && <span className="error">{errors.weight}</span>}
         <br />
+
         <button type="submit">Update Pet</button>
-        <button type="button" onClick={handleCancel}>
-          {' '}
-          Cancel{' '}
+        <button
+          type="button"
+          onClick={() => setIsDeleteModalOpen(true)} // Open delete modal
```

Comment:
What is modal? If I might ask

Reply:
> What is modal? If I might ask

It's basically just a popup window that is displayed on top of the page.